takzero/src/search/node/mod.rs

```diff
@@ -38,6 +38,25 @@ impl<E: Environment> Default for Node<E> {
     }
 }
 
+struct PV<'a, E: Environment> {
+    node: &'a Node<E>,
+}
+
+impl<'a, E: Environment> Iterator for PV<'a, E> {
+    type Item = E::Action;
+
+    fn next(&mut self) -> Option<Self::Item> {
+        let (best_action, best_child) = self
+            .node
+            .children
+            .iter()
+            .max_by_key(|(_, child)| child.visit_count())?;
```

Comment:
Please use `node.select_best_action()` instead since it also handles solved nodes correctly.

Reply:
Fixed, I didn't realize `node.select_best_action()` selects based on visit count for unsolved nodes.